### PR TITLE
ISY-618: Incomplete date widget also accepts normal date entry with dot

### PR DESCRIPTION
**Pull request by mtpham-collab**

### CODE REVIEW

**Review by msg-tbrtz — CHANGES_REQUESTED** [review]
I would wish for a test case that tests the whole chain:
Input a full date string (e.g. 01.01.2020) and test if that is the result.

**Review by msg-tbrtz — CHANGES_REQUESTED** [review]
This doesn't really test a fully typed date. Only the dots are typed, the numbers are set programmaticly.
A component that ignores pressed numbers would still pass this test.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # 17.1.0 -
 ## Fixes
+- ISY-618: Incomplete-Date-Widget akzeptiert auch normale Dateneingabe mit Punkt
```
Comment (msg-tbrtz) [inline CHANGELOG.md:3]: slightly unclear for the user what the error was and what is happening now

**Review by msg-tbrtz — APPROVED** [review]

**Review by msg-tbrtz — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mtpham-collab) [reply CHANGELOG.md]: Change log is updated

**Pull request merged**